scanwise/src/main/java/com/appwise/scanner/managers/PermissionManager.kt

```diff
@@ -0,0 +1,81 @@
+package com.appwise.scanner.managers
+
+import android.app.Activity
+import android.content.pm.PackageManager
+import androidx.activity.ComponentActivity
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+
+object PermissionManager {
```

Comment:
Do we want it to be public? Or do we want to have everything internal to the library and the only point of entry for the user is via the CameraManager? 🤔